.github/workflows/conda_cron.yaml

```diff
@@ -0,0 +1,65 @@
+name: "conda_cron"
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+  push:
+    branches:
+      - main
+  schedule:
+    # At 07:00 UTC every day
+    - cron: "0 7 * * *"
+
+concurrency:
+  group: "${{ github.workflow }}-${{ github.ref }}"
+  cancel-in-progress: true
+
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  condacheck:
+    runs-on: ${{ matrix.OS }}
+    name: "daily conda check"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest', 'macos-13']
+        python-version:
+          - "3.9"
+          - "3.10"
+          - "3.11"
+          - "3.12"
+
+    steps:
+      - name: Setup Micromamba
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-name: openfe
+          create-args: >-
+            python=${{ matrix.python-version }}
+            pip
+          condarc: |
+            channels:
+              - jaimergp/label/unsupported-cudatoolkit-shim
+              - conda-forge
+          init-shell: bash
+
+      - name: "Install openfe + optional deps"
+        run: |
+          micromamba install openfe pytest -c conda-forge
+          python -m pip install pytest-xdist PyGithub
+
+      - name: "env info"
+        run: |
+          micormamba info
+          micromamba list
+          pip list
+
+      - id: run-tests
+        name: "Run tests"
+        run: |
+          # note: this only runs the fast tests
+          pytest -n auto --pyargs openfe openfecli
```

Comment:
I'm relying on a normal failure here rather than raising the issue as we do with gufe. The plan will be to hook up Swenson's cron job checker tool.